### PR TITLE
Add a dry-run default CLI option and a configuration to enable the co…

### PR DESCRIPTION
**Pull request by maximemenager**
…nservation job

### CODE REVIEW

File: cozy-registry.example.yml [hunk 1]
```diff
@@ -45,6 +45,7 @@ swift:
 # Conservation configuration allows to manage paramaters of how many app versions
 # of each channel must be kept during cleaning tasks
 conservation:
+  enabled: false # Specifies if the app cleaning task is enabled
```
Comment (nono) [inline cozy-registry.example.yml]: It is not clear to me if `conservation.enabled` means that the jobs for cleaning will be run, or not.

Comment (nono) [reply cozy-registry.example.yml]: And is this parameter also used for manual executions?

Comment (maximemenager) [reply cozy-registry.example.yml]: > It is not clear to me if `conservation.enabled` means that the jobs for cleaning will be run, or not.

`conservation.enabled = false` means the job won't be executed. Maybe `conservation.remove_enabled` would be a better name ? 

> And is this parameter also used for manual executions?

This parameter is only intended to be used by background jobs, not CLI.

Comment (nono) [reply cozy-registry.example.yml]: Maybe `conservation.enable_background_cleaning`

Comment (maximemenager) [reply cozy-registry.example.yml]: Yep, better :)

**Pull request merged**